tests/Test Applications/Microsoft.Bot.Builder.Twilio.Sample/README.md

```diff
@@ -0,0 +1,86 @@
+# EchoBot using Twilio Adapter
+
+Bot Framework v4 echo bot using Twilio Adapter sample.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to create a simple echo bot that connects with Twilio to respond SMS messages.
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 2.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+## To try this sample
+
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-dotnet.git
+    ```
+
+- In a terminal, navigate to `test/Test Applications/Microsoft.Bot.Builder.Twilio.Sample`
+- Connect the bot with Twilio following the instructions below.

```

Comment:
I think `Connect the bot with Twilio by following` sounds more natural

Reply:
Thanks!